iron-menu-behavior.html

```diff
@@ -55,17 +55,62 @@
 
     listeners: {
       'focus': '_onFocus',
-      'keydown': '_onKeydown'
+      'keydown': '_onKeydown',
+      'iron-items-changed': '_onIronItemsChanged'
     },
 
     keyBindings: {
       'up': '_onUpKey',
       'down': '_onDownKey',
       'esc': '_onEscKey',
-      'enter': '_onEnterKey',
       'shift+tab:keydown': '_onShiftTabDown'
     },
 
+    attached: function() {
+      this._resetTabindices();
```

Comment:
I want to stick with the version as written, since it matches the format of the native API (also the attribute is `tabindex`, not `tab-index`).. but if you're still raw about it, I'll be waiting.

![http://st.ilfattoquotidiano.it/wp-content/uploads/2014/07/Kill-bill-640.jpg](http://st.ilfattoquotidiano.it/wp-content/uploads/2014/07/Kill-bill-640.jpg)